catalog/ui/catalog-ui-search/src/main/webapp/component/content/content.less

```diff
@@ -40,4 +52,14 @@
     vertical-align: top;
     overflow: hidden;
   }
-}
\ No newline at end of file
+
+  .guest-login-btn {
+    background-color: @signInButton;
+    color: #FFF;
+    border: 2.5px solid #1F3A4D;
+    padding: 4px 10px 4px 10px;
+    margin: 0px 20px 0 30px;
+    border-radius: 5px;
```

Comment:
❗️ This CSS feels fragile and likely won't scale well with changes in theme scaling settings. I'm sure we could be using `@minimumButtonSize` or similar vars in here.

Reply:
Updated in 754e4d8